lib/pages/bluehost/bluehost-install-wordpress-details-page.js

```diff
@@ -0,0 +1,24 @@
+import { By as by } from 'selenium-webdriver';
+import config from 'config';
+import BaseContainer from '../../base-container';
+import * as driverHelper from '../../driver-helper';
+
+export default class BluehostInstallWordPressDetailsPage extends BaseContainer {
+	constructor( driver, visit = false ) {
+		const url = config.get( 'bluehostInstallWordPressUrl' );
+		super( driver, by.css( 'form.mm-inputs input[name="ss_site_title"]' ), visit, url );
+	}
+
+	submitAdminDetails() {
+		const siteTitle = config.get( 'bluehostWPInstallSiteTitle' );
+		const username = config.get( 'bluehostWPInstallUsername' );
+		const email = config.get( 'bluehostWPInstallEmail' );
+		const password = config.get( 'bluehostWPInstallPassword' );
```

Comment:
Maybe move these config settings into a common object for easier access?  Not necessary, but would make the NODE_CONFIG more legible.  It could also include the config parameters you call in your other page objects.

```
"bluehostWPInstallDetails": {
  "url", "abc.d.com",
  "siteTitle": "abc",
  "username": "abc",
  "email": "abc@d.com",
  "password": "zyx"
}
```

And then move the whole thing into the constructor so you can pick up the URL.